django/test/client.py

```diff
@@ -589,6 +630,13 @@ def _base_scope(self, **request):
         )
         return scope
 
+    @staticmethod
+    def _transform_headers(headers):
+        return {
+            header_name.replace("-", "_").upper(): value
+            for header_name, value in headers.items()
+        }
```

Comment:
again, only used in one place, inline it

Reply:
So `HttpHeaders.to_header_name()` was added for the WSGI case, which I've now suggested we call `to_wsgi_name()` and likely also move the dictionary comprehension into a `to_wsgi_names()`. I think this could also be added as `HttpHeaders.to_asgi_names()`.